src/App.tsx

```diff
@@ -61,6 +67,11 @@ export default class App extends React.Component <IProps, IState> {
     render() {
         return (
             <div className="App">
+                <Hud
+                    playerStats={this.state.playerStats}
+                    week={this.state.week}
+                />
+                <img src={placeholderImg} alt="Event illustration"/>
```

Comment:
Maybe for this, we check if the current event has a placeholder image first (with `img() === ""`) and only then should be use the placeholder image?

Reply:
Done